nodes/geometry/point_inside_camera_frustrum.py

```diff
@@ -0,0 +1,42 @@
+import bpy
+from bpy.props import *
+from ... base_types.node import AnimationNode
+import bpy_extras
+
+class PointInCameraFrustrum(bpy.types.Node, AnimationNode):
+    bl_idname = "an_PointInCameraFrustrum"
+    bl_label = "Point in Camera Frustrum"
+    
+    def create(self):
+        self.newInput("Scene", "Scene", "scene", hide = True)
+        self.newInput("Object", "Camera", "camera")
+        self.newInput("Vector", "3d Point", "point", value=(0, 0, 0))
+        self.newInput("Float", "Threshold % (positive = interior)", "threshold", value = 0.0)
```

Comment:
Hmm, the socket name should really just be the name. What exactly does this threshold do?